README.md

```diff
@@ -306,6 +306,39 @@ psReadValues("SELECT ... FROM ...", connection, arg1, arg2);
 Note: This function is made only for reading single row. In case you are reading more than one row,
 an *UnexpectedRowCountError* exception is thrown.
 
+**psQuery**
+```c++
+int myData = 0;
+
+psQuery(
+    "SELECT ?",
+    connection,
+    [&](int &value) -> bool {
+        value = myData;
+        return (myData++) < 5; // Return true, if data are set, false otherwise (no more input data available)
+    },
+    [&](int value) {
+        printf("Got value: %d\n", value);
+    }
+)
+```
+
+Or without result callback
+
+```c++
+int myData = 0;
+
+psQuery(
```

Comment:
I'd prefer to use different name for that, not overload.

Reply:
And what name do you suggest? Something like `psWriteQuery`?

Reply:
That implies `INSERT`. I'd suggest to have `psParamQuery` and `psResultQuery` (psReadQuery) as special cases of `psQuery`.